### PR TITLE
android/*: add Ukrainian translation

### PR DESCRIPTION
**Pull request by eternityduck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.uk/android/dalvikvm.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Віртуальна машина Android Java.
+> Більше інформації: <https://source.android.com/devices/tech/dalvik>.
+
+- Запустити конкретну Java програму:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```
Comment (kbdharun) [inline pages.uk/android/dalvikvm.md:8]:
```suggestion
`dalvikvm -classpath {{шлях/до/файлу.jar}} {{ім'я класу}}`
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

File: pages.uk/android/dumpsys.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`dumpsys -l`
+
+- Вивести специфічні аргументи для сервісу:
+
+`dumpsys {{service}} -h`
+
+- Виключити специфічний сервіс з виводу діагностики:
+
+`dumpsys --skip {{service}}`
```
Comment (kbdharun) [inline pages.uk/android/dumpsys.md:25]:
```suggestion
`dumpsys {{сервіс}}`

- Вивести усі сервіси `dumpsys`, про які може бути надана інформація:

`dumpsys -l`

- Вивести специфічні аргументи для сервісу:

`dumpsys {{сервіс}} -h`

- Виключити специфічний сервіс з виводу діагностики:

`dumpsys --skip {{сервіс}}`
```

File: pages.uk/android/getprop.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> Виводить інформацію про системні властивості Android.
+> Більше інформації: <https://manned.org/getprop>.
+
+- Виводить інформацію про системні властивості Android:
+
+`getprop`
+
+- Виводить інформацію про специфічну системну властивість:
+
+`getprop {{property}}`
```
Comment (kbdharun) [inline pages.uk/android/getprop.md:12]:
```suggestion
`getprop {{власність}}`
```

File: pages.uk/android/wm.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+> Ця команда може бути виконана тільки за допомогою `adb shell`.
+> Більше інформації: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Вивести фізичний розмір екрану Android девайсу:
+
+`wm {{size}}`
+
+- Вивести фізичну щільність екрану Android девайсу:
+
+`wm {{density}}`
```
Comment (kbdharun) [inline pages.uk/android/wm.md:13]:
```suggestion
`wm {{розмір}}`

- Вивести фізичну щільність екрану Android девайсу:

`wm {{щільність}}`
```

File: pages.uk/android/screencap.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Зробіть знімок мобільного екрану.
+> Ця команда може бути виконана тільки за допомогою `adb shell`.
+> Більше інформації: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Зробіть знімок мобільного екрану:
+
+`screencap {{path/to/file}}`
```
Comment (kbdharun) [inline pages.uk/android/screencap.md:9]:
```suggestion
`screencap {{шлях/до/файлу}}`
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

File: pages.uk/android/pm.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`pm list packages -s`
+
+- Вивести всі встановлені сторонні (3d-party) застосунки:
+
+`pm list packages -3`
+
+- Вивести застосунки, які підпадають під специфічні ключові слова:
+
+`pm list packages {{keyword1 keyword2 ...}}`
```
Comment (kbdharun) [inline pages.uk/android/pm.md:20]:
```suggestion
`pm list packages {{ключове слово1 ключове слово2 ...}}`
```

File: pages.uk/android/pkg.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`pkg uninstall {{package}}`
+
+- Перевстановити пакет:
+
+`pkg reinstall {{package}}`
+
+- Пошук пакету:
+
+`pkg search {{package}}`
```
Comment (kbdharun) [inline pages.uk/android/pkg.md:24]:
```suggestion
`pkg install {{пакунок}}`

- Видалити пакет:

`pkg uninstall {{пакунок}}`

- Перевстановити пакет:

`pkg reinstall {{пакунок}}`

- Пошук пакету:

`pkg search {{пакунок}}`
```

File: pages.uk/android/logcat.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`logcat --regex {{regular_expression}}`
+
+- Вивести логи для специфічного процесу (PID):
+
+`logcat --pid={{pid}}`
+
+- Вивести логи для процесу специфічного пакету:
+
+`logcat --pid=$(pidof -s {{package}})`
```
Comment (kbdharun) [inline pages.uk/android/logcat.md:24]:
```suggestion
`logcat -f {{шлях/до/файлу}}`

- Вивести рядки, які підпадають під регулярний вираз:

`logcat --regex {{регулярний_вираз}}`

- Вивести логи для специфічного процесу (PID):

`logcat --pid={{pid}}`

- Вивести логи для процесу специфічного пакету:

`logcat --pid=$(pidof -s {{пакунок}})`
```

File: pages.uk/android/input.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+> Ця команда може бути виконана тільки за допомогою `adb shell`.
+> Більше інформації: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
+
+- Відправити код події для одного знаку на Android девайс:
+
+`input keyevent {{event_code}}`
+
+- Відправити текст на Android девайс (`%s` визначає пробіли):
+
+`input text "{{text}}"`
```
Comment (kbdharun) [inline pages.uk/android/input.md:13]:
```suggestion
`input text "{{текст}}"`
```

File: pages.uk/android/input.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# input
+
+> Відправляє коди подій чи жести на сенсорному екрані на Android девайс.
+> Ця команда може бути виконана тільки за допомогою `adb shell`.
+> Більше інформації: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
+
+- Відправити код події для одного знаку на Android девайс:
+
+`input keyevent {{event_code}}`
```
Comment (kbdharun) [inline pages.uk/android/input.md:9]:
```suggestion
`input keyevent {{код_події}}`
```

File: pages.uk/android/input.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+`input tap {{x_position}} {{y_position}}`
+
+- Відправити жест проведення(свайп) на Android девайс:
+
+`input swipe {{x_start}} {{y_start}} {{x_end}} {{y_end}} {{duration_in_ms}}`
+
+- Відправити довге натискання на Android девайс, використовуючи жест проведення(свайп):
+
+`input swipe {{x_position}} {{y_position}} {{x_position}} {{y_position}} {{duration_in_ms}}`
```
Comment (kbdharun) [inline pages.uk/android/input.md:25]:
```suggestion
`input tap {{позиція_x}} {{позиція_y}}`

- Відправити жест проведення(свайп) на Android девайс:

`input swipe {{x_старт}} {{y_старт}} {{х_кінець}} {{y_кінець}} {{тривалість_в_мс}}`

- Відправити довге натискання на Android девайс, використовуючи жест проведення(свайп):

`input swipe {{позиція_x}} {{позиція_y}} {{позиція_x}} {{позиція_y}} {{тривалість_в_мс}}`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Note**: We generally translate applicable placeholders to facilitate native readers to better understand the examples. I added suggestions for the same in this PR (Feel free to reword any of my suggestions, as I am not a native speaker. I used Google translate and your descriptions for reference).

**Tip**: You can batch all of my suggestions in the "Files changed" tab and make a single commit.

File: pages.uk/android/getprop.md (hunk 3 above)
Comment (eternityduck) [reply pages.uk/android/getprop.md:12]: Wrong translation here, will update separately

File: pages.uk/android/logcat.md (hunk 8 above)
Comment (eternityduck) [reply pages.uk/android/logcat.md:24]: Same here, wrong translation, will update separately

**Comment by eternityduck** [comment]
@kbdharun Updated with some of your suggestions and my own updates

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**